configuration_version.go

```diff
@@ -98,19 +98,25 @@ type CVStatusTimestamps struct {
 	StartedAt  time.Time `jsonapi:"attr,started-at,rfc3339"`
 }
 
+// A list of relations to include. See available resources:
+// https://www.terraform.io/docs/cloud/api/configuration-versions.html#available-related-resources
+type ConfigurationVersionIncludeOps string
+
+const (
+	ConfigurationVerIngressAttributes ConfigurationVersionIncludeOps = "ingress_attributes"
+)
+
 // ConfigurationVersionReadOptions represents the options for reading a configuration version.
 type ConfigurationVersionReadOptions struct {
-	Include string `url:"include"`
+	Include []ConfigurationVersionIncludeOps `url:"include"`
```

Comment:
I'm in favor of just a slice, not a pointer to a slice here. See previous comment

Reply:
I have not converted to a pointer because this Include field was never defined as pointer to string, unlike other include fields within go-tfe where they were intentionally (though inconsistently) defined as pointers. So I can address your suggestion in a different PR but for now, in this PR, I rather stick to one breaking change at a time 😊